cmd/prometheus/main.go

```diff
@@ -871,7 +881,7 @@ func main() {
 			name: "gcm_export",
 			reloader: func(cfg *config.Config) error {
 				// Call in closure to not call Global() before it's initialized below.
-				return gcm_export.Global().ApplyConfig(cfg)
+				return gcm_export.Global().ApplyConfig(cfg, nil)
```

Comment:
question: Don't you need to pass in the `exporterOpts` on reload instead of `nil`?

Reply:
It's optional but I added it. Thanks!